Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -919,7 +926,7 @@ namespace AzToolsFramework
 
         InspectorLayout layout = GetCurrentInspectorLayout();
 
-        if (layout == InspectorLayout::LEVEL)
+        if (!m_prefabsAreEnabled && layout == InspectorLayout::OpenPrefabContainerEntity)
```

Comment:
I'm a bit confused by this. When will prefabs be disabled but there will be focus mode?

Reply:
Yeah, this class is getting a little messy because we have a single Inspector widget for both Prefabs and Slices.
In Slices, we instantiate a separate duplicate of the Inspector (the Level Inspector) that always targets the level. In that case, we use the same view that I'm now using for the FocusedPrefabContainerEntity.
I wanted to change the name away from LevelEntity as it was before, since it's now broader, but I can't really use prefab lingo. Maybe I can change it to RootEntity?

Reply:
Tbh, I can't think of any good common term for 'level inspector' and 'focusedPrefabContainerEntity'. What is unique about this layout? Maybe that will give you a better name.

Reply:
It doesn't display the transform component, and it only lets you add level components :D

Reply:
Then InspectorLayout::HideTransform or InspectorLayout::LevelComponentsOnly maybe better

But does that mean that for focused prefab instances, it's possible to add level components?

Reply:
Yes. In the long term, it sounds like we would end up changing the concept of level components to be container components, but they would only be active if applied to the root prefab container, and disabled in all other containers.

Discussing this made me realize that, while that is the long-term objective, we probably want to have a hybrid state for the short term (open prefab container that isn't a level container) that hides the transform, but does not allow level components to be added. When slices are deprecated and we set up the system that would disable "level components" on non-root prefab containers, then we could just merge them... but right now it's just going to create unnecessary confusion and possibly bugs.

So yeah I think I will revert the name of this layout to Level, and just add a new one :) That should also simplify the checks, and allow me to use the editor focus instead of the prefab focus.

Reply:
Done, I ended up keeping the prefab focus checks instead of editor focus, but now it's 2 separate layouts and selection enums.